change/@fluentui-react-datepicker-compat-881179ec-abf2-4f4f-8a61-1b88fd0220e5.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29587&commitId=8278e50788a110d0c2db7969d7ea2c573059bed8&env=prod&clientType=FLUENTUI)


  <details><summary><b>DatePicker Compat</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DatePicker Compat.DateRange- day.chromium.png|0| Added|
|DatePicker Compat.DateRange- month - Dark Mode.chromium.png|0| Added|
|DatePicker Compat.DateRange- month - High Contrast.chromium.png|0| Added|
|DatePicker Compat.DateRange- month.chromium.png|0| Added|
|DatePicker Compat.DateRange- week.chromium.png|0| Added|
|DatePicker Compat.DateRange- work week.chromium.png|0| Added|
|DatePicker Compat.DateRange- day.default.chromium.png|0| Removed|
|DatePicker Compat.DateRange- month - Dark Mode.default.chromium.png|0| Removed|
|DatePicker Compat.DateRange- month - High Contrast.default.chromium.png|0| Removed|
|DatePicker Compat.DateRange- month.default.chromium.png|0| Removed|
|DatePicker Compat.DateRange- week.default.chromium.png|0| Removed|
|DatePicker Compat.DateRange- work week.default.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>